src/routes/recipe.ts

```diff
@@ -38,4 +40,21 @@ router.get(
   validate,
   getRecipesLikeByUserId
 );
+router.delete(
+  "/:id",
+  protectedRoute,
+  getParamIdValidator(),
+  validate,
+  (req, res) => {
```

Comment:
Typescript will complain if it can't determine the types of the arguments to the function. In this case it's this express function: `(req, res)`

Tell typescript what the type is for `req` and the type for `res` using the colon syntax.

```suggestion
  (req: IRequest, res: Response) => {
```

Reply:
Does recipe.id need to be added src/routes/definitions.ts?
recipe: {
       id: string;
}

Reply:
Hey I am not quite understanding the question

Reply:
I am using recipe.id in the route but in  src/routes/definitions.ts, Irequest doesn't define recipe.id 

import { Request } from "express";
export interface IRequest extends Request {
  user: {
    id: string;
  };
  recipe: {
    id: string
  };
}